src/client/pfs/pfs.proto

```diff
@@ -79,6 +79,11 @@ message RepoAuthInfo {
   auth.Scope access_level = 1;
 }
 
+// These are the different places where a commit may be originated from
+enum CommitOrigin {
+  MAKE = 0;
+  PROPAGATE = 1;
+}
```

Comment:
Could we make `CommitOrigin` into a message and then have an enum inside it called `Op` or something that describes which operation created the commit? We're going to want to add a few things to this struct over time, such as the user that created the commit and this will make that easy to do. Also, I don't think `MAKE` and `PROPAGATE` are going to be clear to users, I was thinking something like `USER` and `AUTO` might be more understandable, although I don't love those.